src/sql/parts/profiler/dialog/profilerColumnEditorDialog.ts

```diff
@@ -402,6 +405,7 @@ export class ProfilerColumnEditorDialog extends Modal {
 		});
 		this._tree.setInput(this._element);
 		this._tree.layout(DOM.getTotalHeight(this._treeContainer));
+		*/
```

Comment:
Commented code here and above. Did you mean to leave it commented out?

Reply:
I added comments to these methods referencing the open GitHub issues for the dialog

Reply:
Thanks for taking care of this and the other comments I left. Everything looks good to me